src/lib/Monads.ml

```diff
@@ -267,7 +267,7 @@ struct
 
   let emit ?(lvl = `Info) loc pp a : unit m =
     fun (st, _env) ->
-    Log.pp_message ~loc ~lvl pp a;
+    Log.pp_message ~loc ~lvl ~last_token:"" pp a;
```

Comment:
I'm too scared to change emit at the moment because I don't understand the refiner... =)